docs/site/Interceptors.md

```diff
@@ -1,9 +1,9 @@
 ---
 lang: en
-title: 'Interceptors'
+title: 'Interceptor'
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Interceptor
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Interceptors.html
+permalink: /doc/en/lb4/Interceptor.html
```

Comment:
I think I understand the rationale behind changing the html path name is to make it consistent with the title. If that's the case, can we keep the same html and add the `redirect_to`? Renaming the html path will cause broken links in other docs or articles (mostly external ones, not part of LB docs). WDYT?

Reply:
@dhmlau 👍 Fair enough, adding.